interfaces.go

```diff
@@ -112,6 +114,8 @@ type MultimapMkTable interface {
 	Update(v, id map[string]interface{}, valuesToUpdate map[string]interface{}) Op
 	Delete(v, id map[string]interface{}) Op
 	DeleteAll(v map[string]interface{}) Op
+	// List populates the provided pointer to a slice with the results matching the keys provided.
```

Comment:
It's interesting that the two List functions has different names for the args, do they act differently? I've always wondered why you'd provide a clustering key to List, but I guess it makes sense if it's a start ID 🤷